torch/csrc/jit/tensorexpr/ir_printer.cpp

```diff
@@ -271,6 +272,9 @@ void IRPrinter::visit(const Load* v) {
     }
     ind->accept(this);
   }
+  if (v->indices().empty()) {
```

Comment:
Nit: should we omit "[0]" for scalar loads/stores all together to distinguish from 1D buffer of size 1?

Reply:
I dont think so, it's still a Tensor.